docs/README.md

```diff
@@ -201,6 +201,52 @@ versions:
   for Linux
 - [`nvm-windows`](https://github.com/coreybutler/nvm-windows) - Works with native Windows
 
+##### Install Git
+Booster will initialize a Git repository when you create a new project, so it is required that you have it already installed it in your system.
+
+###### Ubuntu
+
+```shell
+sudo apt install git-all
+```
+
+###### macOS
+
+```shell
+brew install git
+```
+
+###### Windows
+
+```shell
+choco install git
+```
+
+###### Git configuration variables
+After installing git in your machine, you need to take a look in the global variables `user.name` and `user.email`, that should be properly configured globally.
+you can can check this page for more informations: [Git configuration variables](https://git-scm.com/docs/git-config) and below you can configure the variables
+
+###### Ubuntu
+
+```shell
+git config --global user.name "Your Name Here"
+git config --global user.email "your_email@youremail.com"
+```
+
+###### macOS
+
+```shell
+git config --global user.name "Your Name Here"
+git config --global user.email "your_email@youremail.com"
+```
+
+###### Windows
+
+```shell
+git config --global user.name "Your Name Here"
+git config --global user.email "your_email@youremail.com"
```

Comment:
Those are all the same steps, so we can just leave them once:

```suggestion
```shell
git config --global user.name "Your Name Here"
git config --global user.email "your_email@youremail.com"
```
```